pom.xml

```diff
@@ -37,22 +37,21 @@
     <properties>
         <!-- Versioning -->
         <version.arquillian.core>1.1.11.Final</version.arquillian.core>
-        <version.com.opera>1.5</version.com.opera>
         <version.phantom.driver>1.2.1.Final</version.phantom.driver>
-        <version.selenium>2.53.1</version.selenium>
-        <version.htmlunit.driver>2.20</version.htmlunit.driver>
+        <version.selenium>3.0.1</version.selenium>
+        <version.htmlunit.driver>2.23</version.htmlunit.driver>
         <version.junit>4.11</version.junit>
```

Comment:
Any breaking changes are definitely more important then testing with old arq-core versions.

> this build will have to take this into consideratiojn

you are right about this :-)

Ok, I'll append a commit with related changes ;-)